gdal/doc/source/drivers/raster/nitf.rst

```diff
@@ -149,6 +149,13 @@ Creation Options:
 -  **FILE_TRE=tre-name=tre-contents**: Similar to above
    options, except that the TREs are written in the file header, instead
    of the image header.
+-  **DES=des-name=des-contents**: One or more DES creation options may
+   be provided to write arbitrary user defined DESs to the NITF file.
+   The des-name should be at most 25 characters, and the des-contents
+   should be "backslash escaped" if it contains backslashes or zero
+   bytes.  The des-contents must contain standard DES fields, starting
```

Comment:
maybe add a reference to the CPLEscapeString(str, -1, CPLES_BackslashQuotable) function

Reply:
done